sdk/agrifood/tests.yml

```diff
@@ -8,6 +8,7 @@ extends:
       Clouds: Public_FarmBeats
       CloudConfig:
         Public_Farmbeats:
+          ServiceConnection: azure-sdk-tests-public #Might need a specific service connection if this is a unique sub.
```

Comment:
@weshaggard I checked and they are using a custom sub (`5bae917c-5fc9-40c5-923d-adbd52759048`). I don't have access though.

Reply:
I don't think they care much about this pipeline currently as the existing client secret is also not working. So, either way they will need to spend time to fix this when they are interested.

Reply:
@weshaggard what is the guidance for services that use a custom sub? Our openai tests use static resources from a non-SDK team owned sub.

Reply:
There is no general guidance. We need to look into them case by case.